frontend/src/components/MeetingRoom/Toolbar/Toolbar.tsx

```diff
@@ -60,57 +60,74 @@ const Toolbar = ({
   const isReportIssueEnabled = import.meta.env.VITE_ENABLE_REPORT_ISSUE === 'true';
   const isViewingScreenShare = subscriberWrappers.some((subWrapper) => subWrapper.isScreenshare);
   const isScreenSharePresent = isViewingScreenShare || isSharingScreen;
-  const isSmallViewport = useIsSmallViewport();
   const handleLeave = useCallback(() => {
     if (!disconnect) {
       return;
     }
     disconnect();
   }, [disconnect]);
   const [openEmojiGridDesktop, setOpenEmojiGridDesktop] = useState<boolean>(false);
+  const shownButtons = useShownButtons();
 
   return (
-    <div className="absolute bottom-0 left-0 flex h-[80px] w-full flex-col items-center bg-darkGray-100 p-4 md:flex-row md:justify-between">
-      <div className="flex flex-1 justify-start overflow-hidden pr-2">
```

Comment:
Moved some of these to the `TimeRoomNameMeetingRoom` component. The right margin is only necessary when it's rendered.